.github/ISSUE_TEMPLATE/prepare_minor_release.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Minor release tracking issue
+title: Prepare for v<release-tag>
+---
+
+Please see this documentation for more details baout release process: https://github.com/metal3-io/metal3-docs/blob/main/processes/releasing.md
+
+
+**Note**:
+* The following is based on the v1.7 minor release. Modify according to the tracked minor release.
+
+## Tasks
+
+* [ ] Add/edit JJBs to acommodate new build jobs and PR jobs for new release branches. [Prior art](https://gerrit.nordix.org/c/infra/cicd/+/20875)
+* [ ] Add Prow jobs for new release branches in Project-infra repo [Prior art](https://github.com/metal3-io/project-infra/pull/697)
+* [ ] Prepare dev-env to acommodate release-1.7 related changes and configurations [Prior art](https://github.com/metal3-io/metal3-dev-env/pull/1381)
```

Comment:
I think we should not put past release versions in the template.

Reply:
removed versions, can you check again